README.md

```diff
@@ -1,9 +1,61 @@
 # Falco Rules
 
-## Notice
+👷‍♀️ **Under active development** 👷‍♂️
 
-This repository has been created upon this [proposal](https://github.com/falcosecurity/falco/blob/master/proposals/20221129-artifacts-distribution.md#move-falco-rules-to-their-own-repo) and is not yet ready for use. The progress tracker can be found [here](https://github.com/falcosecurity/evolution/issues/229).
+[![License](https://img.shields.io/github/license/falcosecurity/plugins?style=for-the-badge)](./LICENSE)
 
-## ⚠️ Current status
+Note: *This repository has been created upon this [proposal](https://github.com/falcosecurity/falco/blob/master/proposals/20221129-artifacts-distribution.md#move-falco-rules-to-their-own-repo) and is not yet ready for use. The progress tracker can be found [here](https://github.com/falcosecurity/evolution/issues/229).*
+
+This repository contains the [Rule Registry](#rule-registry) and the [rules officially maintained](#falcusecurity-rules) by the Falcosecurity organization. [Rules](https://falco.org/docs/rules) tell [Falco](https://github.com/falcosecurity/falco) what to do. Please refer to the [official documentation](https://falco.org/docs/rules) to better understand the rules' concepts. 
+
+## Rule Registry
+
+The Registry contains metadata and information about every rule known and recognized by the Falcosecurity organization. These rules are developed for Falco and made available to the community. Check out the sections below to know how to [register your rules](#registering-a-new-rule) and see rules currently contained in the registry.
+
+### Registering a new Rule
+
+Registering your rule inside the registry helps ensure that some technical constraints are respected. Moreover, this is a great way to share your rule project with the community and engage with it, thus gaining new users and **increasing its visibility**. We encourage you to register your rule in this registry before publishing it.
```

Comment:
```suggestion
Registering your rule inside the registry helps ensure that some technical constraints are respected. Moreover, this is a great way to share your ruleset and make it available to the community. We encourage you to register your ruleset in this registry before publishing it.
```

Reply:
We don't know yet if this repository is open to any community-curated ruleset or if it's reserved only for Falco official rules files. So I'd avoid setting wrong expectations at the moment.

Reply:
Which means that we can leave out the entire "registering a new rule" section for the moment but leave the registry format as documentation?